server/src/main/java/org/opensearch/indices/replication/checkpoint/TransportCheckpointInfoResponse.java

```diff
@@ -11,38 +11,45 @@
 import org.opensearch.common.io.stream.StreamInput;
 import org.opensearch.common.io.stream.StreamOutput;
 import org.opensearch.index.store.Store;
+import org.opensearch.index.store.StoreFileMetadata;
 import org.opensearch.indices.replication.copy.ReplicationCheckpoint;
 import org.opensearch.transport.TransportResponse;
 
 import java.io.IOException;
+import java.util.Set;
 
 public class TransportCheckpointInfoResponse extends TransportResponse {
 
     private final ReplicationCheckpoint checkpoint;
     private final Store.MetadataSnapshot snapshot;
     private final byte[] infosBytes;
+    private final Set<StoreFileMetadata> additionalFiles;
```

Comment:
Nitpick - might be worth it to comment what `additionalFiles` is expected to contain